eox_tenant/signals.py

```diff
@@ -53,7 +53,10 @@ def _update_settings(domain):
     LOG.debug("PID: %s CONFIGURING THE SETTINGS OBJECT | %s", os.getpid(), tenant_key)
     for key, value in six.iteritems(config):
         if isinstance(value, dict):
-            merged = getattr(base_settings, key, {}).copy()
+            try:
+                merged = getattr(base_settings, key, {}).copy()
+            except AttributeError:
```

Comment:
It is not necessary to add an attribute error, because if the attribute does not exist, getattr always returns the default value. @felipemontoya

Reply:
It is true that getattr returns the default, but the error we are catching here comes from the fact that not always `base_settings.key` will have a .copy attribute

Reply:
I have found cases where the production data will override a string with a dict

Reply:
Ok, i didn't realize that.